tools/update_roles.php

```diff
@@ -0,0 +1,642 @@
+<?php
+/**
+ * Script to add or remove permissions from a role
+ *
+ * PHP Version 5
+ *
+ * @category Main
+ * @package  Loris
+ * @author   Tara Campbell <tara.campbell@gmail.com>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris
+ */
+
+set_include_path(get_include_path().":".__DIR__."/../php/libraries:");
+require_once __DIR__ . "/../vendor/autoload.php";
+
+$client = new NDB_Client();
+$client->makeCommandLine();
+$client->initialize(__DIR__."/../project/config.xml");
+
+$roles       = getRoles();
+$permissions = getPermissions();
+
+// User asking for the list of roles
+if (isset($argv[1]) && $argv[1] === "roles") {
+    if (!empty($roles)) {
+        print "The roles in the database are:\n\n";
+        prettyPrint($roles);
+    } else {
+        print "There are currently no roles in the database.\n";
+    }
+    
+    exit();
+}
+
+// User asking for the list of permissions
+if (isset($argv[1]) && $argv[1] === "permissions") {
+    print "The permissions in the database are:\n\n";
+    prettyPrint($permissions);
+    exit();
+}
+
+// User asking to reassign roles based on the user permissions
+if (isset($argv[1]) && $argv[1] === "refresh") {
+    refresh();
+    exit();
+}
+
+if (isset($argv[1]) && !isRole($argv[1])) {
+    syntaxIncorrect();
+    exit();
+}
+
+$role = $argv[1];
```

Comment:
`PHP Notice:  Undefined offset: 1 in /var/www/loris/tools/update_roles.php on line 54`

when update_roles.php called with no arguments (to get the usage examples printout)